bot/exts/info/source.py

```diff
@@ -83,12 +95,12 @@ async def build_embed(self, source_object: SourceType) -> Optional[Embed]:
         elif isinstance(source_object, commands.Command):
             description = source_object.short_doc
             title = f"Command: {source_object.qualified_name}"
-        elif isinstance(source_object, TagIdentifier):
-            title = f"Tag: {source_object}"
-            description = ""
-        else:
+        elif issubclass(type(source_object), commands.Cog):
             title = f"Cog: {source_object.qualified_name}"
             description = source_object.description.splitlines()[0]
+        else:
+            title = f"Tag: {source_object}"
+            description = ""
 
         embed = Embed(title=title, description=description)
         embed.add_field(name="Source Code", value=f"[Go to GitHub]({url})")
```

Comment:
This currently doesn't work for Tags because it returns the wrong path

Reply:
![image](https://user-images.githubusercontent.com/48383734/223869528-93e19747-5732-4442-8c36-528599f4066d.png)

Notice how the `bot` directory is missing, which results in a 404 when clicking on the guild.

Also, would be great it we could stay consitent with slashes instead of anti slashes for paths

Reply:
Fixed in https://github.com/python-discord/bot/pull/2446/commits/758a99417ea102edd726fd8f7167a4dbeaade556